pkgs/tools/compression/gzip/default.nix

```diff
@@ -15,10 +15,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ xz.bin ];
 
-  # In stdenv-linux, prevent a dependency on bootstrap-tools.
-  makeFlags = "SHELL=/bin/sh GREP=grep";
-
-  doCheck = false; # fails
```

Comment:
The line might perhaps serve as documentation why checks aren't done, but no strong opinion from men on that.